opm/output/eclipse/EclipseIO.hpp

```diff
@@ -53,19 +53,53 @@ class EclipseIO {
 
 
 
-    /// Write the static eclipse data (grid, PVT curves, etc) to disk, and set
-    /// up additional initial properties
-    ///
-    /// - simProps contains a list of properties which must be
-    ///   calculated by the simulator, e.g. the transmissibility
-    ///   vectors TRANX, TRANY and TRANZ.
-    ///
-    /// - The NNC argument is distributed between the EGRID and INIT
-    ///   files.
-    ///
-    /// If you want FOE in the SUMMARY section, you must pass the initial
-    /// oil-in-place "OIP" key in Solution
-    void writeInitial( data::Solution = data::Solution(), const NNC& = NNC());
+    /*
```

Comment:
Just a minor thing: this changes the doc string from being a doxygen doc string to just a regular comment. Was that intended?